src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../App.css';
+
+const Calculator = () => (
+  <div className="main">
+    <div className="calculator">
+      <div className="screen">0</div>
+      <div className="input-container">
+        <div className="inputs">AC</div>
+        <div className="inputs">+/-</div>
+        <div className="inputs">%</div>
+        <div className="inputs-right">/</div>
+        <div className="inputs">7</div>
+        <div className="inputs">8</div>
+        <div className="inputs">9</div>
+        <div className="inputs-right">x</div>
+        <div className="inputs">4</div>
+        <div className="inputs">5</div>
```

Comment:
- [ ] Please ensure that the `Calculator` component has at least one child component as specified in the project requirements 👍